packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist.res

```diff
@@ -168,6 +168,15 @@ let pdf = (f: float, t: t) => {
 let inv = T.Integral.yToX
 let cdf = T.Integral.xToY
 
+
+@genType
+let toSparkline = (buckets: int, t: t ): string => {
+  let size : float = T.maxX(t) -. T.minX(t)
+  let stepSize = size /. Belt.Int.toFloat(buckets)
+  let cdfImage = E.A.rangeFloat(~step=stepSize, T.minX(t), T.maxX(t)) -> Belt.Array.map(val => cdf(val,t))
+  Sparklines.create(E.A.diff(cdfImage), ())
```

Comment:
I don't quite get the above two lines. Are you getting the CDF points, then getting the derivative of them, to get the PDF? Why not just get the PDF directly?

Reply:
I was considering using PDF, but then I don't think discrete or mixed distributions would work?

Reply:
In that case, it should be more clear what you're doing (at least with comments about that process). 

Now that I think about it, I feel like the above 5 lines would be a fair bit more elegant if you could achieve this by making a continuous PDF with a pointLength of N. (It's already basically the right format, no need to manually redo this with stepsize). 

In order for it to work with mixed distributions, there should be a function to make all dists continuous. This could be by converting them to cdfs and back to pdfs. (This functionality exists)

Reply:
I had a go at this, it looks fine! Only thing is that it seems uniform distributions don't respect pointSetDistLength